common/src/app/common/types/tokens_lib.cljc

```diff
@@ -590,6 +589,14 @@
       (check-token-theme-attrs)
       (map->TokenTheme)))
 
+(defn make-token-theme
+  [& {:as attrs}]
+  (make-token-theme-impl attrs))
+
+(defn make-theme
+  [attrs]
+  (make-token-theme-impl attrs))
+
```

Comment:
This is not necessary. In clojure you can define a function with keyword arguments and call it with a single map:

https://clojure.org/news/2021/03/18/apis-serving-people-and-programs

In fact, other `make-xxx` functions in this module are being used this way.

Reply:
Wow, I didn't know that actually, thanks!
I'll revert it back to the original signature.